framework/Models/PostProcessor.py

```diff
@@ -175,6 +175,8 @@ def createNewInput(self,myInput,samplerType,**kwargs):
            a mandatory key is the sampledVars'that contains a dictionary {'name variable':value}
       @ Out, myInput, list, the inputs (list) to start from to generate the new one
     """
+    if 'createNewInput' in dir(self._pp):
+      myInput = self._pp.createNewInput(myInput,samplerType,**kwargs)
```

Comment:
should we remove the "samplerType" (I know it was present already)? Postprocesessors do not use sampleType or we are thinking here to extend the possibility to have PP behave differently depending on the Sampler that generated the data? (e.g. Sortino -> Sobol, OtherSamplers -> least square of HDMR coefficients, etc,)?

Reply:
Considering also Paul's comment, I suggest to change the function name "createNewInput(myInput,samplerType,**kwargs)" to something like "processInput(myInput, **kwargs)". The "samplerType" will be removed.

Reply:
As long as the specific kwargs are listed in the base Interface definition of `createNewInput`, I think this would be simple to maintain.

Reply:
changed to use "createPostprocessorInput" and "samplerType" is removed.